UglyTicTacToe/UglyTicTacToe/Game.aspx.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace UglyTicTacToe
+{
+    public enum Player
+    {
+        None,
+        X,
+        O
+    }
```

Comment:
Nice use of an enumeration! Since this enumeration is used in two different pages, I'd put it into its own `Player.cs` file.